test/unit/Sync.spec.ts

```diff
@@ -0,0 +1,163 @@
+    `;
+    flicking1 = new Flicking(wrapper1);
+  });
+
+  afterEach(() => {
+    cleanup();
+    flicking0.destroy();
+    flicking1.destroy();
+  });
+
+  it("main flicking should move with other flickings", async () => {
+    // Given
+    flicking0.addPlugins(new Sync({
+      type: "camera",
+      synchronizedFlickingOptions: [
+        {
+          flicking: flicking0
+        },
+        {
+          flicking: flicking1
+        }
+      ]
+    }));
+    await waitEvent(flicking0, "ready");
+
+    // When
+    void flicking0.control.moveToPosition(200, 0);
+
+    // Then
+    expect(flicking1.camera.position).not.to.equal(100);
+  });
+
+  it("other flickings should move main flicking", async () => {
+    // Given
+    flicking0.addPlugins(new Sync({
+      type: "camera",
+      synchronizedFlickingOptions: [
+        {
+          flicking: flicking0
+        },
+        {
+          flicking: flicking1
+        }
+      ]
+    }));
+    await waitEvent(flicking1, "ready");
+
+    // When
+    void flicking1.control.moveToPosition(200, 0);
+
+    // Then
+    expect(flicking0.camera.position).not.to.equal(100);
+  });
+
+  it("active panel should have active class", async () => {
+    // Given
+    flicking0.addPlugins(new Sync({
+      type: "index",
+      synchronizedFlickingOptions: [
+        {
+          flicking: flicking0,
+          isSlidable: true
+        },
+        {
+          flicking: flicking1,
+          activeClass: "flicking-thumbnail-active"
+        }
+      ]
+    }));
+    await waitEvent(flicking0, "ready");
+
+    // When
+    void flicking0.moveTo(1, 0);
+    tick(100);
+
+    // Then
+    expect(flicking1.getPanel(0).element.className.includes("flicking-thumbnail-active")).equal(false);
+    expect(flicking1.getPanel(3).element.className.includes("flicking-thumbnail-active")).equal(true);
+  });
+
+  it("slidable flicking should move other flickings", async () => {
+    // Given
+    flicking0.addPlugins(new Sync({
+      type: "index",
+      synchronizedFlickingOptions: [
+        {
+          flicking: flicking0,
+          isSlidable: true
+        },
+        {
+          flicking: flicking1
+        }
+      ]
+    }));
+    await waitEvent(flicking0, "ready");
+
+    // When
+    void flicking0.moveTo(1, 0);
+    tick(100);
+
+    // Then
+    expect(flicking1.camera.position).not.to.equal(100);
+  });
+
+  it("clickable flicking should move other flickings", async () => {
+    // Given
+    flicking0.addPlugins(new Sync({
+      type: "index",
+      synchronizedFlickingOptions: [
+        {
+          flicking: flicking0,
+          isClickable: true
+        },
+        {
+          flicking: flicking1
+        }
+      ]
+    }));
+    await waitEvent(flicking0, "ready");
+
+    // When
+    flicking0.trigger(new ComponentEvent("select", { index: 2, panel: flicking0.getPanel(2), direction: null }));
+    tick(100);
+
+    // Then
+    expect(flicking1.camera.position).not.to.equal(100);
```

Comment:
Same, I think you should also check the index here.

Reply:
Added index check for tests. Resolved with comment above.